_posts/2023-04-09-haberdashers-win-sr2023.md

```diff
@@ -0,0 +1,110 @@
+
+Check out the [rulebook][rules] for all the details on the awards we give.
+
+## Thank You
+
+This year’s competition would not have been possible without all of our amazing sponsors. Their generous support allows us to make Student Robotics free to enter and help us continue in our mission to bring the excitement of engineering and the challenge of coding to young people through robotics.
+
+### Our SR2023 Sponsors
+
+- [The University of Southampton Engineering and Computer Science](https://ecs.soton.ac.uk/)
+- [Adventurous Machines](https://adventurousmachines.com/?ref=studentrobotics)
+- [Mythic Beasts](https://www.mythic-beasts.com/)
+- [ITDev](https://www.itdev.co.uk/)
+- [Bittele](https://www.7pcb.com/)
+- [Hexibox](https://hexibox-events.co.uk/)
+
+We’d also like to thank our volunteers, who make Student Robotics happen every year! Some have helped at the competition itself, while others have been in teams working throughout the year to organise the event. Our Competition Team designs, organises, and delivers Tech Days and the Competition weekend. Our Kit Team designs and supports the software and hardware our competitors use; the Infrastructure Team ensures that our website stays up and our internal teams can work collaboratively; the Fundraising Team ensures that we have the resources needed to run our events; and the Marketing Team makes sure our efforts are seen and heard by all. A huge thank you to everyone who contributed to the success of this year’s competition.
+
+If you’re reading this and want to join us next year, sign up on our [volunteering page][volunteers].
+
+## Notes to editors
+
+Student Robotics is an annual robotics competition for 16-19 year-olds in the UK
+and Europe. It was founded in 2006 by university students and is free to enter
+thanks to our [sponsors][sponsor] and [many volunteers][volunteers]. Since it
+was first run in 2008, the final competition has grown from one room at the
+University of Southampton[^1] to the UK's biggest autonomous robotics
+competition.
+
+[^1]: Student Robotics is independent from the University of Southampton.
+
+At the start of the academic year, teams are given a kit containing custom-made
+electronics at a Kickstart event, where the game for the year is announced. They
+then have until the Easter holiday to build fully-autonomous robots, which will
+compete against each other in the final competition. They are supported by
+volunteer mentors, and software to assist them in programming their robots is
+provided.
+
+If you would like to find out more, please [get in touch][contactus].
+
+_The SR Team_
+
+[volunteers]: {{ '/volunteer' | prepend: site.baseurl }}
+[contactus]: mailto:pr@studentrobotics.org
+[rules]: https://studentrobotics.org/docs/rules/
```

Comment:
Nit: In time, this rules link will stop linking to the correct version. We should make a permalink archive of the rules and link to those instead.

Reply:
```suggestion
[rules]: https://studentrobotics.org/docs/resources/2023/rulebook.html
```